src/Compilers/Core/CodeAnalysisTest/Analyzers/AnalyzerFileReferenceTests.cs

```diff
@@ -750,5 +750,6 @@ public void AddDependencyLocation(string fullPath) { }
         public bool IsHostAssembly(Assembly assembly) => false;
         public Assembly LoadFromPath(string fullPath) => throw new Exception();
         public string? GetOriginalDependencyLocation(AssemblyName assembly) => throw new Exception();
+        public void Dispose() { }
```

Comment:
This change was extracted into https://github.com/dotnet/roslyn/pull/74845 and will go away once that pr is in.